src/NuGet.Indexing/DownloadsByVersion.cs

```diff
@@ -48,8 +41,11 @@ public int Total
 
             set
             {
+                int oldValue;
```

Comment:
are the updates done on a single thread? In any case, there should be a comment about that